lib/http.js

```diff
@@ -5,6 +5,7 @@ var storageUtil = require('./storageUtil');
 var Q = require('q');
 var AuthApiError = require('./errors/AuthApiError');
 var config = require('./config');
+var $ = require('jquery');
```

Comment:
Not sure about this change. Only a few files have a dependency on jquery

Reply:
We'd like to keep auth-js small and jquery is a pretty big dependency, so if a dev wants to use jquery, they have to do it explicitly.

For example, normally, okta-auth-js is imported like this: `var oktaAuth = require('@okta/okta-auth-js')`. This uses a small library called `reqwest` to make http requests. If a dev wants to use jQuery to makes requests instead, they can import okta-auth-js like this: `var oktaAuth = require('@okta/okta-auth-js/jquery')`.

Reply:
👍